### DISCUSSION TITLE
How can i change the Populate in users/me

### DISCUSSION THREAD

**Discussion by sidahmedabdelillah** [strapi/strapi#7231] [Legacy Topics]
How can i change the Populate in users/me

**Comment by lauriejim** [comment]
Hello! To do that you will have to customize the `me` function of the **Users & Permissions** plugin.
Here is the function - https://github.com/strapi/strapi/blob/master/packages/strapi-plugin-users-permissions/controllers/User.js#L116
In this function we can see it returns the `ctx.state.user` data. But this data is populated by the permission check function.
I suggest here to use the same code as the findOne function but to for the `id` to be `ctx.state.user.id`.
To customize, you will have to check the customization concept into the docs.
https://strapi.io/documentation/v3.x/concepts/customization.html
